app/src/main/java/com/azuresamples/msalandroidapp/MainActivity.java

```diff
@@ -22,7 +22,7 @@
 public class MainActivity extends AppCompatActivity {
 
     /* Azure AD v2 Configs */
-    final static String SCOPES [] = {"https://graph.microsoft.com/User.Read"};
+    final static String[] SCOPES = {"https://graph.microsoft.com/User.Read"};
```

Comment:
MS Graph scopes do not need to be prefixed

Reply:
@iambmelt gave the opposite feedback - this initialization is more C than Java style he taught me. I usually do what you have above as well :).